README.md

```diff
@@ -42,7 +42,7 @@ Then, update your project's pom.xml file dependencies, as follows:
   <dependency>
       <groupId>com.yahoo.imapnio</groupId>
       <artifactId>imapnio.core</artifactId>
-      <version>4.2.5</version>
+      <version>4.2.6</version>
```

Comment:
Should we bump to 4.3 and change the minor.minor version?

Reply:
done